public/app/features/alerting/unified/components/receivers/grafanaAppReceivers/types.ts

```diff
@@ -0,0 +1,23 @@
+import { Receiver } from '../../../../../../plugins/datasource/alertmanager/types';
+// we will add in here more types if needed
+export enum GrafanaAppReceiverEnum {
+  GRAFANA_ONCALL = 'GrafanaOnCall',
```

Comment:
I think it's better to show it as "Grafana OnCall"

```suggestion
  GRAFANA_ONCALL = 'Grafana OnCall',
```